### PR TITLE
Removed encrypt at rest control for Storage as it's by default behaviour, changed the message type  for OMS data pushing error 

### PR DESCRIPTION
**Pull request by jaslokunal**
- Removed encrypt at rest control for Storage as it's by default behaviour 
- Changing message type for error while pushing data to OMS

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/SubscriptionCore/SubscriptionCore.ps1 [hunk 1]
```diff
@@ -477,7 +477,7 @@ class SubscriptionCore: SVTBase
 		{
 			$controlResult.AddMessage([MessageData]::new("Security center policies must be configured with settings mentioned below:", $this.SecurityCenterInstance.Policy.properties));			
 
-			if(($this.MisConfiguredASCPolicies | Measure-Object).Count -ne 0 -and $this.SecurityCenterInstance.IsLatestVersion)
```
Comment (ganesh-msft) [inline src/AzSK/Framework/Core/SVT/SubscriptionCore/SubscriptionCore.ps1]: Have you removed the IsLatestVersion check altogether??

Comment (jaslokunal) [reply src/AzSK/Framework/Core/SVT/SubscriptionCore/SubscriptionCore.ps1]: No, in else part we are checking for it

Comment (sbyna-msft) [reply src/AzSK/Framework/Core/SVT/SubscriptionCore/SubscriptionCore.ps1]: @jaslokunal  Why did you need to do this change?

Comment (jaslokunal) [reply src/AzSK/Framework/Core/SVT/SubscriptionCore/SubscriptionCore.ps1]: Azure_Subscription_Config_Azure_Security_Center  control is passing even though we have mis-configured ASC policies if SecurityCenterConfigVersion is not latest

**Pull request merged**